zephyr/modules/owntech_data_api/zephyr/public_api/DataAPI.cpp

```diff
@@ -110,11 +110,38 @@ void DataAPI::setParameters(channel_t channel, float32_t gain, float32_t offset)
 	data_conversion_set_conversion_parameters_linear(channel_info.adc_num, channel_info.channel_num, gain, offset);
 }
 
+float32_t DataAPI::getParameter(channel_t channel, parameter_t parameter_name)
```

Comment:
I think this function name should be more expressive, such as `retreiveParameterStoredValue`.